verticaltabs.js

```diff
@@ -560,6 +556,11 @@ VerticalTabs.prototype = {
       browserPanel.insertBefore(bottom, document.getElementById('fullscreen-warning').nextSibling);
       top.palette = palette;
       this.window.TabsInTitlebar.updateAppearance(true);
+
+      tab_context_menu.removeEventListener('mouseover', enter);
+      tab_context_menu.removeEventListener('mouseout', exit);
+      toolbar_context_menu.removeEventListener('mouseover', enter);
+      toolbar_context_menu.removeEventListener('mouseout', exit);
```

Comment:
but... the context menus stick around....they get reused elsewhere, so I have to remove the listeners. (Actually, I wonder if that means opening a context menu on the tab bar above would cause TC to open...shoot!)